webapp/portlet/src/main/webapp/vue-apps/version-history-drawer/components/VersionHistoryDrawer.vue

```diff
@@ -0,0 +1,141 @@
+<!--
+  This file is part of the Meeds project (https://meeds.io/).
+  Copyright (C) 2022 Meeds Association
+  contact@meeds.io
+  This program is free software; you can redistribute it and/or
+  modify it under the terms of the GNU Lesser General Public
+  License as published by the Free Software Foundation; either
+  version 3 of the License, or (at your option) any later version.
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+  Lesser General Public License for more details.
+  You should have received a copy of the GNU Lesser General Public License
+  along with this program; if not, write to the Free Software Foundation,
+  Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+-->
+<template>
+  <exo-drawer
+    ref="versionHistoryDrawer"
+    class="versionHistoryDrawer"
+    @closed="closed"
+    show-overlay
+    right>
+    <template slot="title">
+      {{ $t('versionHistory.label.title') }}
+    </template>
+    <template slot="content">
+      <div
+        v-if="isLoading"
+        class="text-center mt-5">
+        <v-progress-circular
+          indeterminate
+          color="primary" />
+      </div>
```

Comment:
```suggestion
```

Use `this.$refs.versionHistoryDrawer.startLoading()` & `this.$refs.versionHistoryDrawer.endLoading()` instead of adding a dedicated loading effect especially to this element. This can be done using a watch:
```javascript
watch: {
  isLoading() {
    if (this.isLoading) {
      this.$refs.versionHistoryDrawer.startLoading();
    } else {
      this.$refs.versionHistoryDrawer.endLoading();
    }
  },
},
```

Reply:
done